test/resources/sampleNodeApp/package.json

```diff
@@ -50,7 +50,7 @@
   },
```

Comment:
These changes are not required, this is just a test project!

Reply:
ack